pytest_doctestplus/plugin.py

```diff
@@ -54,67 +43,110 @@ def __exit__(self, *args, **kwargs):
 # these pytest hooks allow us to mark tests and run the marked tests with
 # specific command line options.
 def pytest_addoption(parser):
-    parser.addoption("--doctest-plus", action="store_true",
-                     help="enable running doctests with additional "
-                          "features not found in the normal doctest "
-                          "plugin")
-
-    parser.addoption("--doctest-rst", action="store_true",
-                     help=(
-                         "Enable running doctests in .rst documentation. "
-                         "This is no longer recommended, use --doctest-glob instead."
-                     ))
-
-    parser.addoption("--text-file-format", action="store",
-                     help=(
-                        "Text file format for narrative documentation. "
-                        "Options accepted are 'txt', 'tex', and 'rst'. "
-                        "This is no longer recommended, use --doctest-glob instead."
-                     ))
+    parser.addoption(
+        '--doctest-plus',
+        action='store_true',
+        help=(
+            "enable running doctests with additional features not found "
+            "in the normal doctest plugin"
+        ),
```

Comment:
these are just the black reformattings I was complaining about. The whole section looses the information that it's belong to `parser.addoption` by doing a custom indent, while adds nothing to readability, etc.

Reply:
oh, it wasn't black's fault. I did this by hands 😅 (at least this concrete example)